var/spack/repos/builtin/packages/snakemake/package.py

```diff
@@ -130,5 +130,6 @@ class Snakemake(PythonPackage):
     )
     depends_on("py-requests", when="+http", type=("build", "run"))
 
-    def test(self):
-        Executable("snakemake")("--version")
+    def test_run(self):
+        """Test if snakemake runs correctly"""
```

Comment:
Nit: There is no check of the output so not sure I'd say `correctly`.  It is definitely a check to see if `snakemake` runs with the version option.